ray-operator/controllers/ray/rayservice_controller.go

```diff
@@ -556,16 +556,30 @@ func (r *RayServiceReconciler) cleanUpRayClusterInstance(ctx context.Context, ra
 		return err
 	}
 
+	// Determine the ray cluster deletion delay seconds
+	deletionDelay := RayClusterDeletionDelayDuration
+	if rayServiceInstance.Spec.RayClusterDeletionDelaySeconds != nil {
+		if *rayServiceInstance.Spec.RayClusterDeletionDelaySeconds < 0 {
```

Comment:
move the check to https://github.com/ray-project/kuberay/blob/master/ray-operator/controllers/ray/utils/validation.go

In addition, we have already checked the value in OpenAPI. Do we still need the check?